### PR TITLE
Update Descheduler profiles enhancement to 'implemented'

### PR DESCRIPTION
**Pull request by damemi**

### CODE REVIEW

**Review by damemi — COMMENTED** [review]
/cc @soltysh @ingvagabund

File: enhancements/scheduling/descheduler-profiles.md [hunk 1]
```diff
@@ -23,10 +23,10 @@ superseded-by:
...
 
 - [x] Enhancement is `implementable`
-- [ ] Design details are appropriately documented from clear requirements
-- [ ] Test plan is defined
-- [ ] Graduation criteria for dev preview, tech preview, GA
-- [ ] User-facing documentation is created in [openshift-docs](https://github.com/openshift/openshift-docs/)
+- [x] Design details are appropriately documented from clear requirements
+- [x] Test plan is defined
+- [x] Graduation criteria for dev preview, tech preview, GA
+- [x] User-facing documentation is created in [openshift-docs](https://github.com/openshift/openshift-docs/)
```
Comment (ingvagabund) [inline enhancements/scheduling/descheduler-profiles.md:29]: Can you put a link to the documentation as well?

Comment (damemi) [reply enhancements/scheduling/descheduler-profiles.md]: Updated the link

**Review by soltysh — APPROVED** [review]
/approve

**Pull request merged**